maps_dashboards/public/components/tooltip/tooltipTable.tsx

```diff
@@ -37,12 +36,27 @@ function mergeTables(tables: Table[], selectedIndex: number[]) {
   const merged: TableData = [];
   const allSelected = selectedIndex.includes(ALL_LAYERS);
 
-  for (let i = 0; i < tables.length; i++) {
-    if (allSelected || selectedIndex.includes(i)) {
-      merged.push(...tables[i].table);
+  if (!allSelected) {
+    for (const index of selectedIndex) {
+      merged.push(...tables[index].table);
     }
+    return merged;
```

Comment:
We don't add layer name if not all items are selected?

Reply:
Yeah. By default, the layer name will be available in title if single layer is selected.